cluster-state-service/internal/features/e2e_stream_instances.feature

```diff
@@ -0,0 +1,8 @@
+@e2e @instance @stream-instances
+Feature: Stream Instances
+
+  Scenario: Get instance stream
+    Given I have some instances registered with the ECS cluster
+    When I stream all instance events and start a task
```

Comment:
I would separate this into 2 steps since they perform 2 completely different actions.
`When I stream all instance events`
`And I start a task`

(Same with the task steps)